ComponentKit/TransactionalDataSources/Common/CKTransactionalComponentDataSourceAppliedChanges.mm

```diff
@@ -48,6 +48,11 @@ - (instancetype)initWithUpdatedIndexPaths:(NSSet *)updatedIndexPaths
   return self;
 }
 
+- (NSString *)description
+{
+  return [NSString stringWithFormat:@"Updated Index Paths: %@\nRemoved Index Paths: %@\nRemove Sections: %@\nMoves: %@\nInserted Sections: %@\nInserted Index Paths: %@", _updatedIndexPaths, _removedIndexPaths, _removedSections, _movedIndexPaths, _insertedSections, _insertedIndexPaths];
```

Comment:
What do you mean by "more standard syntax"? 

Will take care of splitting onto new lines.